src/pay/batch_payer.py

```diff
@@ -513,47 +607,60 @@ def attempt_single_batch(self, payment_records, op_counter, dry_run=None):
 
         content_list = []
 
-        total_gas = total_fees = 0
+        total_gas = total_tx_fees = total_burn_fees = 0
 
-        for payment_item in payment_records:
+        for payment_item in payment_items:
 
             pymnt_amnt = payment_item.amount  # expected in micro tez
-            storage_limit, gas_limit, tx_fee = (
+            payment_item.transaction_fee = (
+                self.default_fee
+            )  # init transaction fee with the default fee
+            # Get initial default values for storage, gas and fees
+            # These default values are used for non-empty tz1 accounts transactions
+            storage_limit, gas_limit, tx_fee, burn_fee = (
                 self.storage_limit,
                 self.gas_limit,
                 self.default_fee,
+                0,
             )
 
             # TRD extension for non scriptless contract accounts
             if payment_item.paymentaddress.startswith("KT"):
                 simulation_status, simulation_results = self.simulate_single_operation(
                     payment_item, pymnt_amnt, branch, chain_id
                 )
+
                 if simulation_status == PaymentStatus.FAIL:
                     logger.info(
                         "Payment to {} script could not be processed. Possible reason: liquidated contract. Skipping. Think about redirecting the payout to the owner address using the maps rules. Please refer to the TRD documentation or to one of the TRD maintainers.".format(
                             payment_item.paymentaddress
                         )
                     )
                     payment_item.paid = PaymentStatus.AVOIDED
+                    payment_item.desc += " Investigate with tzkt.io: Liquidated oven or no default entry point. Use rules map for payment redirect."
                     continue
+
                 gas_limit, tx_fee, storage_limit = simulation_results
                 burn_fee = COST_PER_BYTE * storage_limit
-                total_fee = tx_fee + burn_fee
+                payment_item.transaction_fee = tx_fee
 
                 if KT1_FEE_SAFETY_CHECK:
+                    total_fee = tx_fee + burn_fee
```

Comment:
Just to make sure so the `total_fee` is not equal to the transaction fee? The transaction_fee is only equal to tx_fee?